assets/js/modules/analytics/datastore/properties.js

```diff
@@ -407,6 +408,53 @@ const baseSelectors = {
 
 		return select( STORE_NAME ).isFetchingGetPropertiesProfiles( accountID );
 	} ),
+
+	/**
+	 * Gets all Analytic and GA4 properties this account can access.
+	 *
+	 * Returns an array of all UA + GA4 analytics properties.
+	 *
+	 * Returns `undefined` if accounts have not yet loaded.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state     Data store's state.
+	 * @param {string} accountID The Analytics Account ID to fetch properties for.
+	 * @return {(Array.<Object>|undefined)} An array of Analytics properties; `undefined` if not loaded.
+	 */
+	getPropertiesIncludingGA4: createRegistrySelector( ( select ) => ( state, accountID ) => {
+		let properties = select( STORE_NAME ).getProperties( accountID );
+
+		if ( select( MODULES_ANALYTICS_4 ) ) {
+			const propertiesGA4 = select( MODULES_ANALYTICS_4 ).getProperties( accountID );
+			properties = properties.concat( propertiesGA4 );
+		}
+
+		const isGA4 = ( property ) => !! property._id;
+		const compare = ( a, b ) => {
+			if ( a < b ) {
+				return -1;
+			}
+			if ( a > b ) {
+				return 1;
+			}
+			return 0;
+		}
```

Comment:
Forgot to update here as well:
```suggestion
		};
```